web/elm/src/Grid.elm

```diff
@@ -1,381 +0,0 @@
-module Grid exposing (Grid(..), MatrixCell(..), fromGraph, height, insert, toMatrix, width)
```

Comment:
This file doesn't seem to have to do with the main change... I assume it's just dead code removal?

Reply:
yeah, we considered it part of the 'cleaning up deps'